requirements/linux-rocm.txt

```diff
@@ -5,7 +5,7 @@ accelerate
 huggingface_hub
```

Comment:
I had to downgrade as well in order to make huggingface model downloads working:

`huggingface_hub==0.22.0`

But I'm not entirely sure if this PR is the right place to do that because I'm pretty sure other platforms are affected as well.

Reply:
I fixed that properly in #810

Reply:
I see, you migrated to the API. That's much better.